joomla/joomla-cms
# [4.0] Add loading=lazy for the custom field media images

Pull Request for Issue # .

### Summary of Changes

Add loading=lazy for the custom field media

### Testing Instructions

Enable Custom fields
Create a custom field of type media

Create an article in a category with this custom field available

Add an image (in the custom field)

Observe in the front end

### Expected result



### Actual result



### Documentation Changes Required

This is a system wide change, meaning that the documentation will reflect all these changes (eg nothing special for this instance)

@wilsonge @laoneo
